homeassistant/helpers/entity_registry.py

```diff
@@ -152,6 +163,58 @@ def hidden(self) -> bool:
         """Return if entry is hidden."""
         return self.hidden_by is not None
 
+    @property
+    def _as_display_dict(self) -> dict[str, Any] | None:
+        """Return a partial dict representation of the entry.
+
+        This version only includes what's needed for display.
+        """
+        if self.disabled_by is not None:
```

Comment:
This feels more suited as a check in the websocket command. This should just return the display dict.

Reply:
If we move the check, we waste cycles on building a display dict which is not used anywhere.